src/interface/heatmap.jl

```diff
@@ -118,6 +122,10 @@ function heatmap(z::AbstractMatrix; xlim = (0., 0.), ylim = (0., 0.), xoffset =
         noextrema = false
     catch
     end
+    if zrange != (0., 0.)
```

Comment:
Just for consistency, maybe `zlim` here?

Reply:
Yeah, that would be more consistent